### PR TITLE
P1- Text Utilites

### PR DESCRIPTION
**Pull request by GarvG**
Completed With  Component of CL Text Utilites
Files to Check 
Text Utilites.html 
Text Utilites.css

### CODE REVIEW

File: Text-Utilies.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (zoyatanweer) [inline Text-Utilies.html:12]: write a comment here as to specify what is this link being used for.

File: Text-Utilies.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=lucario&wt=none&l=auto&width=819.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522smalltext%2522%253E%250A%2520%2520%253Ch1%253E%253Csmall%253E...%253C%252Fsmall%253E%253C%252Fh1%253E%250A%2520%2520%253Ch2%253E%253Csmall%253E...%253C%252Fsmall%253E%253C%252Fh2%253E%250A%2520%2520%253Ch3%253E%253Csmall%253E...%253C%252Fsmall%253E%253C%252Fh3%253E%250A%2520%2520%253Ch4%253E%253Csmall%253E...%253C%252Fsmall%253E%253C%252Fh4%253E%250A%2520%2520%253Ch5%253E%253Csmall%253E...%253C%252Fsmall%253E%253C%252Fh5%253E%250A%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+                    style="width: 820px; height: 354px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                </div>
+
+                <div class="center-container">
+                    <h3 class="headers"> Radndom Text</h3>
```
Comment (zoyatanweer) [inline Text-Utilies.html]: Random spelling wrong.

File: components.css [hunk 3]
```diff
@@ -43,7 +44,7 @@ body{
     display: block;
     height: 100%;
     width: 100%;
-    line-height: 65px;
+    line-height: 50px;
```
Comment (zoyatanweer) [inline components.css]: this value can be in rem

File: components.css [hunk 4]
```diff
@@ -18,6 +18,7 @@
     --primarycolor:#35589A;
```
Comment (zoyatanweer) [inline components.css:18]: use var for color.

**Review by zoyatanweer — APPROVED** [review]
code structure is good. UI is very neat. Just make the suggested changes and you're good to merge. Nice work!

File: components.css [hunk 5]
```diff
@@ -43,7 +44,7 @@ body{
     display: block;
     height: 100%;
     width: 100%;
-    line-height: 65px;
+    line-height: 50px;
     font-size: 1.2rem;
     color: black;
```
Comment (parulgupta2001) [inline components.css:49]: use var for color

File: index.html [hunk 6]
```diff
@@ -25,6 +25,7 @@
                 <li><a href="card.html">CARD</a></li>
                 <li><a href="Images.Html">IMAGE</a></li>
                 <li><a href="Input.html">INPUT</a></li>
+                <li><a href="Text-Utilies.html">Text-Utilies</a></li>
```
Comment (parulgupta2001) [inline index.html:28]: correct the spelling of "utilities"

**Review by parulgupta2001 — APPROVED** [review]
Everything looks ok, only consider the suggestions given below.

File: styles.css [hunk 7]
```diff
@@ -6,6 +6,7 @@
 @import url(card.css);
 @import url(images.css);
 @import url(input.css);
+@import url(Text-Utilies.css);
```
Comment (sudipkundu999) [inline styles.css:9]: follow a consistent naming convention.
`text-utilities.css` would be better here

File: components.css [hunk 8]
```diff
@@ -18,6 +18,7 @@
     --primarycolor:#35589A;
     --ButtonLink:#99FEFF;
     --background-card:white;
+    --textgray:gray;
```
Comment (sudipkundu999) [inline components.css:21]: Suggestion: Since you're using hex before for declaring color variables, you may use it here as well for consistency.

Comment (GarvG) [reply components.css:21]: will take care from next time

File: styles.css (hunk 7 above)
Comment (GarvG) [reply styles.css:9]: will take care from next time

File: index.html (hunk 6 above)
Comment (GarvG) [reply index.html:28]: typing error will take care from next time

File: Text-Utilies.css [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+{
+    color:var(--textgray);
+    font-size: 2rem;
+}
+.Underlined
+{
+    text-decoration: underline;
+    font-size: 2rem;
+}
+.Strike
```
Comment (sudipkundu999) [inline Text-Utilies.css:18]: The naming convention is inconsistent.
Refer this: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

File: components.css (hunk 5 above)
Comment (GarvG) [reply components.css:49]: was using this color only once so used it directly

**Review by sudipkundu999 — APPROVED** [review]
The code looks good. 
Just a consistent naming convention needs to be followed.
Good to merge with changes.

File: components.css (hunk 3 above)
Comment (GarvG) [reply components.css]: Done

File: Text-Utilies.html (hunk 2 above)
Comment (GarvG) [reply Text-Utilies.html]: Done

File: Text-Utilies.html (hunk 1 above)
Comment (GarvG) [reply Text-Utilies.html:12]: Done

File: Text-Utilies.css (hunk 9 above)
Comment (GarvG) [reply Text-Utilies.css:18]: Will take Care from next Time Thank you  for the Suggestion Really Appreciated

**Pull request merged**